ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -0,0 +1,169 @@
+'''
+Tests for pforma_io
+'''
+import unittest
+import os
+import sys
+import shutil
+import tempfile
+from mock import patch
+import logging
+from StringIO import StringIO
+from contextlib import contextmanager
+from ph5.clients import ph5tostationxml
+from ph5.utilities import kef2ph5
+from ph5 import logger, ch as CH
+from ph5.core import experiment
+
+
+@contextmanager
+def captured_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
+
+def log_capture_string():
+    capture = StringIO()
+    logger.removeHandler(CH)
+    ch = logging.StreamHandler(capture)
+    logger.addHandler(ch)
+
+    return capture
+
+
+def initialize_ph5(nickname, path='.', editmode=False):
+    ex = experiment.ExperimentGroup(nickname=nickname, currentpath=path)
+    ex.ph5open(editmode)
+    ex.initgroup()
+    return ex
+
+
+class TestPH5toStationXMLParser(unittest.TestCase):
+    def getParser(self, level, minlat=None, maxlat=None, minlon=None,
+                  maxlon=None, lat=None, lon=None, minrad=None, maxrad=None):
+        self.ph5sxml = [ph5tostationxml.PH5toStationXMLRequest(
+            minlatitude=minlat,
+            maxlatitude=maxlat,
+            minlongitude=minlon,
+            maxlongitude=maxlon,
+            latitude=lat,
+            longitude=lon,
+            minradius=minrad,
+            maxradius=maxrad
+        )]
+        self.mng = ph5tostationxml.PH5toStationXMLRequestManager(
+            sta_xml_obj_list=self.ph5sxml,
+            ph5path=".",
+            nickname="master.ph5",
+            level=level,
+            format="TEXT"
+        )
+        return ph5tostationxml.PH5toStationXMLParser(self.mng)
+
+    def createPH5(self, path, arraykef):
+        kef2ph5.EX = ex = initialize_ph5("master.ph5", path, True)
+        kef2ph5.PH5 = path + "/master.ph5"
+        kef2ph5.TRACE = False
+
+        kef2ph5.KEFFILE = self.home + arraykef
+
+        kef2ph5.populateTables()
+        # add experiment_t
+        kef2ph5.KEFFILE = self.home + \
+            "/ph5/test_data/metadata/experiment.kef"
+        kef2ph5.populateTables()
+        ex.ph5close()
+
+    def setUp(self):
```

Comment:
Please move setUp and tearDown to the top of the class.

Reply:
I just move setUp and tearDown to the top of the class.